### PR TITLE
Cherry pick 2.9.2

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Description
 - [x] https://issues.redhat.com/browse/MGDAPI-2441 - commits 1-4
 - [x] https://issues.redhat.com/browse/MGDAPI-2278 - commits 6-9
 - [x] https://issues.redhat.com/browse/MGDAPI-2253 - commits 6-9
 - [x] https://issues.redhat.com/browse/INTLY-10768 - commits 6-9
 - [x] https://issues.redhat.com/browse/MGDAPI-2313 - commit 5
 - [x] https://issues.redhat.com/browse/INTLY-10773 - commit 15-16
 - [x] https://issues.redhat.com/browse/INTLY-10775 - commit 11-13

## Verification

- Install OSD 4.7.21
- Installed RHMI 2.9.1 using index -> quay.io/laurafitzgerald/integreatly-index:2.9.1
- Upgraded to RHMI 2.9.2 using index -> quay.io/laurafitzgerald/integreatly-index:2.9.2
      - CRO version is 0.30.0 
      - AMO version is 1.6.2 
      - Verify UPS is version 0.5.1  and alerts are using the correct queries as shown in this [pr](https://github.com/aerogear/unifiedpush-operator/pull/105)
      - Follow the verification steps from https://github.com/integr8ly/integreatly-operator/pull/2012 to verify that all alerting and dashboards are working as expected
- Upgrade to 4.8
     - Verify that the UPS alert isn't firing
     -  Verify 3Scale and OAuth routes don't have `router-default.` in them as described in https://github.com/integr8ly/integreatly-operator/pull/2003
     -  Follow the verification steps from https://github.com/integr8ly/integreatly-operator/pull/2012 to verify that all alerting and dashboards are working as expected

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: apis/v1alpha1/rhmi_types.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ var (
 	VersionMonitoring          ProductVersion = "1.6.0"
```
Comment (laurafitzgerald) [inline apis/v1alpha1/rhmi_types.go:88]: @Patryk-Stefanski can you update this to 1.6.2

File: apis/v1alpha1/rhmi_types.go [hunk 2]
```diff
@@ -120,7 +120,7 @@ var (
 	OperatorVersionCodeReadyWorkspaces OperatorVersion = "2.1.1"
 	OperatorVersion3Scale              OperatorVersion = "0.7.0"
 	OperatorVersionFuse                OperatorVersion = "1.6.0"
-	OperatorVersionCloudResources      OperatorVersion = "0.29.0"
+	OperatorVersionCloudResources      OperatorVersion = "0.30.0"
 	OperatorVersionUPS                 OperatorVersion = "0.5.0"
 	OperatorVersionApicurioRegistry    OperatorVersion = "0.0.3"
 	OperatorVersionApicurito           OperatorVersion = "1.6.0"
```
Comment (laurafitzgerald) [inline apis/v1alpha1/rhmi_types.go:126]: also here

File: manifests/integreatly-monitoring/1.6.2/application-monitoring-operator.v1.6.2.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+  - Grafana
+  - Monitoring
+  - AlertManager
+  maintainers:
+  - email: placeholder@email.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: application-monitoring-operator.v1.6.1
```
Comment (laurafitzgerald) [inline manifests/integreatly-monitoring/1.6.2/application-monitoring-operator.v1.6.2.clusterserviceversion.yaml:353]:
```suggestion
replaces: application-monitoring-operator.v1.6.0
  skips:
    - application-monitoring-operator.v1.6.1
  
```

**Comment by laurafitzgerald** [comment]
e2e tests are failing before getting to the tests themselves
```level=info msg="Cluster operator monitoring Available is False with : "
level=fatal msg="failed to initialize the cluster: Cluster operator monitoring is still updating"``` from install-install pod

**Comment by psturc** [comment]
Completed the verification after upgrading OSD to 4.8.2
I also ran automated tests and there was one test failing

```
C04 - Verify Alerts exist
...
Missing Rules: [UnifiedPushJavaHeapThresholdExceeded]
Unexpected Rules: []
Status: File found with missing or unexpected rules
```

Not sure what could have caused this. Let's see if e2e tests are passing

**Comment by laurafitzgerald** [comment]
Failed check is now resolved. The code has been verified functionally and on the various versions of OSD that were required. Tests are all passing so merging.

**Pull request merged**